dusty/commands/setup.py

```diff
@@ -84,6 +85,13 @@ def _get_boot2docker_vm_size():
     else:
         return _get_raw_input('Please input the number of megabytes to allocate to the vm: ')
 
+def _check_ssh_add():
+    output = ''
+    try:
+        check_call_demoted(['sh', '-c', 'ssh-add -l'])
+    except:
+        log_to_client('\nYou must run `ssh-add -K` in order to add your ssh keys to your keychain. Dusty relies on this to pull repos. \n')
```

Comment:
You do not need to give the path to the key generally.  If they have a more complicated ssh config setup i am assuming they would know how to do more complicated commands with ssh-add